upgrade-test/colony-upgrade.js

```diff
@@ -27,6 +27,8 @@ contract("Colony contract upgrade", accounts => {
   let updatedColony;
   let updatedColonyVersion;
 
+  const dueDate = 112233445566;
```

Comment:
Is this a sticky key or something? What's wrong which `currentBlockTime`?

Reply:
Yeah, don't know what I was thinking here.